bdns/dns.go

```diff
@@ -179,6 +190,7 @@ func NewDNSResolverImpl(readTimeout time.Duration, servers []string, stats metri
 		dnsClient:                dnsClient,
 		servers:                  servers,
 		allowRestrictedAddresses: false,
+		caaSERVFAILExceptions:    caaSERVFAILExceptions,
```

Comment:
Did you mean to make this comment somewhere else? This code passes through the exception list as passed to the constructor. If we just ran make here, we'd lose that information.